llvm-external-projects/iree-compiler-api/python/CompilerModule.cpp

```diff
@@ -57,6 +57,26 @@ class PyFileAccumulator {
 
 }  // namespace
 
+static const char BUILD_MHLO_IMPORT_PASS_PIPELINE_DOCSTRING[] =
+    R"(Populates MHLO import passes on a PassManager.
+
+This enables standalone access to the import pipeline that can be run as part
+of main compilation with the `set_input_dialect_mhlo()` option. It is provided
+seprately to facilitate integration with frontend workflows.
+
+This pipeline requires IREE-compatible MHLO input: MHLO control flow must have
+been legalized to SCF or CFG and tuples must not exist. See the
```

Comment:
We can have a pass to enforce this :slightly_smiling_face: that would provide clear error messages and a well-defined boundary